### PR TITLE
Add Lidar USB Viewer

### PR DESCRIPTION
**Pull request by ecourtois**
Refactoring: prepare support of more tools than just simulator.

Add new tool to visualize data provided by a Lidar connected on an USB port via a Serial-USB adapter.

### CODE REVIEW

**Review by Kheprys — APPROVED** [review]
LGTM

File: cogip/tools/lidarusb/lidarwidget.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+from PySide2.QtCharts import QtCharts
+from PySide2.QtCore import Slot as qtSlot
+
+from cogip.tools.lidarusb import datastruct
+from cogip.tools.lidarusb.tablemodel import TableModel
+
+
+class LidarWidget(QtWidgets.QWidget):
+    """
+    This class is a widget that provides the graphic view and instantiate the polar chart, sliders and the table view.
```
Comment (gdoffe) [inline cogip/tools/lidarusb/lidarwidget.py:11]: instantiates

File: cogip/tools/lidarusb/lidarwidget.py [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+    Attributes:
+        min_distance: min value of the distance range (use on the chart)
+        max_distance: max value of the distance range (use on the chart)
+        max_intensity: min value of the intensity range (use on the chart)
```
Comment (gdoffe) [inline cogip/tools/lidarusb/lidarwidget.py:16]: max or min ?

**Review by gdoffe — CHANGES_REQUESTED** [review]
Please sign off your commit

File: cogip/tools/lidarusb/lidarwidget.py (hunk 1 above)
Comment (ecourtois) [reply cogip/tools/lidarusb/lidarwidget.py]: Done

File: cogip/tools/lidarusb/lidarwidget.py (hunk 2 above)
Comment (ecourtois) [reply cogip/tools/lidarusb/lidarwidget.py]: It is max, done.

**Review by gdoffe — APPROVED** [review]

**Pull request merged**